### PR TITLE
add release notes for december 4 release

### PR DESCRIPTION
**Pull request by lukasvogel**

### CODE REVIEW

File: data/releasenotes.json [hunk 1]
```diff
@@ -1,4 +1,26 @@
 [
+	{
+		"version": "alpha/2024-12-05",
+		"date": "2024-12-05",
+		"breaksdb": false,
+		"notes": [
+			"<b>New Feature:</b> The docker image is now compatible with extrnal <tt>pg_isready</tt> health check. Previously, CedarDB would be started and stopped multiple times during setup while being externally reachable. This caused <tt>pg_isready</tt> to prematurely declare CedarDB being up and subsequent connections to fail.",
```
Comment (MoritzS) [inline data/releasenotes.json]:
```suggestion
			"<b>New Feature:</b> The docker image is now compatible with external <tt>pg_isready</tt> health check. Previously, CedarDB would be started and stopped multiple times during setup while being externally reachable. This caused <tt>pg_isready</tt> to prematurely declare CedarDB being up and subsequent connections to fail.",
```

File: data/releasenotes.json [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+			"<b>New Feature:</b> Sequence names now automatically convert to oids. You can now write <tt>CREATE SEQUENCE serial; select nextval('serial');</tt>",
+			"<b>New Feature:</b> Log messages are now prefixed with a timestamp.",
+			"<b>New Feature:</b> More detailed CSV import error messages: When the format is wrong, CedarDB now returns the offending column and the detailed issue.",
+			"<b>New Feature:</b> CedarScript now supports Common Table Expressions.",
+			"<b>PostgreSQL compatibility:</b> Support for <tt>factorial(n)</tt>, in addition to the already existing <tt>n!</tt> and <tt>!!n</tt>.",
+			"<b>PostgreSQL compatibility:</b> Subtracting two dates now returns an integer with the difference in days, instead of an interval.",
+			"<b>PostgreSQL compatibility:</b> <tt>split_part</tt> now allows for negative arguments to search backward from the end and correctly returns <tt>null</tt> in some corner cases.",
```
Comment (MoritzS) [inline data/releasenotes.json:13]:
```suggestion
			"<b>PostgreSQL compatibility:</b> <tt>split_part</tt> now allows for negative arguments to search backwards from the end and correctly returns <tt>null</tt> in some corner cases.",
```

**Review by MoritzS — APPROVED** [review]

File: data/releasenotes.json [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+	{
+		"version": "alpha/2024-12-05",
+		"date": "2024-12-05",
+		"breaksdb": false,
+		"notes": [
+			"<b>New Feature:</b> The docker image is now compatible with extrnal <tt>pg_isready</tt> health check. Previously, CedarDB would be started and stopped multiple times during setup while being externally reachable. This caused <tt>pg_isready</tt> to prematurely declare CedarDB being up and subsequent connections to fail.",
+			"<b>New Feature:</b> Sequence names now automatically convert to oids. You can now write <tt>CREATE SEQUENCE serial; select nextval('serial');</tt>",
+			"<b>New Feature:</b> Log messages are now prefixed with a timestamp.",
+			"<b>New Feature:</b> More detailed CSV import error messages: When the format is wrong, CedarDB now returns the offending column and the detailed issue.",
+			"<b>New Feature:</b> CedarScript now supports Common Table Expressions.",
```
Comment (ChrisWint) [inline data/releasenotes.json]:
```suggestion
```
1st rule of CedarScript, don't talk about CedarScript

File: data/releasenotes.json [hunk 4]
```diff
@@ -1,4 +1,26 @@
 [
+	{
+		"version": "alpha/2024-12-05",
+		"date": "2024-12-05",
+		"breaksdb": false,
+		"notes": [
+			"<b>New Feature:</b> The docker image is now compatible with extrnal <tt>pg_isready</tt> health check. Previously, CedarDB would be started and stopped multiple times during setup while being externally reachable. This caused <tt>pg_isready</tt> to prematurely declare CedarDB being up and subsequent connections to fail.",
+			"<b>New Feature:</b> Sequence names now automatically convert to oids. You can now write <tt>CREATE SEQUENCE serial; select nextval('serial');</tt>",
```
Comment (ChrisWint) [inline data/releasenotes.json]: Is this really a new feature or rather compatibility?

**Review by ChrisWint — APPROVED** [review]

**Pull request merged**